src/amo/sagas/categories.js

```diff
@@ -5,25 +5,23 @@ import { hideLoading, showLoading } from 'react-redux-loading-bar';
 import { call, put, select, takeEvery } from 'redux-saga/effects';
 /* eslint-enable import/order */
 
-import {
-  categoriesFail,
-  categoriesLoad,
-} from 'core/actions/categories';
+import { categoriesLoad } from 'core/actions/categories';
 import { categories as categoriesApi } from 'core/api';
 import { CATEGORIES_FETCH } from 'core/constants';
 import log from 'core/logger';
-import { getApi } from 'core/sagas/utils';
+import { createErrorHandler, getApi } from 'core/sagas/utils';
 
 
-export function* fetchCategories() {
+export function* fetchCategories({ payload: { errorHandlerId } }) {
+  const errorHandler = createErrorHandler(errorHandlerId);
   try {
     yield put(showLoading());
     const api = yield select(getApi);
     const response = yield call(categoriesApi, { api });
     yield put(categoriesLoad(response));
-  } catch (err) {
-    log.warn('Categories failed to load:', err);
-    yield put(categoriesFail(err));
+  } catch (error) {
```

Comment:
Thanks, I think short variables names are silly. 👍  (If I did this, which I think I did, I was wrong.)